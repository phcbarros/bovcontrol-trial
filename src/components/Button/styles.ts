import styled from 'styled-components/native'

export const Container = styled.TouchableOpacity`
  width: 100%;
  height: 56px;
  background-color: #8257e5;
  justify-content: center;
  align-items: center;
`

export const Title = styled.Text`
  color: #fff;
  font-size: 16px;
  font-weight: bold;
`
