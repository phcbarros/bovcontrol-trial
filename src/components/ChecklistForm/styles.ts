import styled from 'styled-components/native'

export const Container = styled.ScrollView.attrs({
  showsVerticalScrollIndicator: false,
  contentContainerStyle: {paddingBottom: 40, justifyContent: 'flex-start'},
})``

export const Form = styled.View`
  width: 100%;
  background-color: ${({theme}) => theme.COLORS.backgroundPrimary};
  padding: 20px;
  gap: 16px;
`
