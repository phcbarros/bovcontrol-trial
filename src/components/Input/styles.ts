import styled from 'styled-components/native'

export const Container = styled.View``

export const Group = styled.View`
  width: 100%;
  height: 56px;
  background-color: #fff;
  flex-direction: row;
  align-items: center;
  overflow: hidden;
`

export const Icon = styled.View`
  width: 56px;
  height: 56px;
  justify-content: center;
  align-items: center;
  overflow: hidden;
  border-right-width: 3px;
  border-right-color: #f4f5f6;
`
export const Control = styled.TextInput`
  flex: 1;
  padding-left: 16px;
  font-size: 16px;
`

export const Error = styled.Text`
  font-size: 14px;
  color: #dc1637;
  margin-top: 8px;
`
