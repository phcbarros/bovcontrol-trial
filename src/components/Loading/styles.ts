import styled from 'styled-components/native'

import theme from '../../styles/theme'

export const Container = styled.View`
  flex: 1;
  align-items: center;
  justify-content: center;

  background-color: ${theme.COLORS.backgroundPrimary};
`

export const LoadIndicator = styled.ActivityIndicator.attrs(() => ({
  color: theme.COLORS.text,
}))``
