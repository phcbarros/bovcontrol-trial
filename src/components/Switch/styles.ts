import styled from 'styled-components/native'

export const Container = styled.View`
  width: 100%;
`

export const Group = styled.View`
  width: 100%;
  height: 56px;
  background-color: #fff;
  flex-direction: row;
  align-items: center;
  justify-content: flex-start;
  overflow: hidden;
`

export const Icon = styled.View`
  width: 56px;
  height: 56px;
  justify-content: center;
  align-items: center;
  overflow: hidden;
  border-right-width: 3px;
  border-right-color: #f4f5f6;
`

export const Control = styled.Switch.attrs({
  thumbColor: '#fff',
  trackColor: {true: '#8257e5', false: '#fff'},
})`
  padding-left: 16px;
`

export const Label = styled.Text`
  font-size: 14px;
  margin-bottom: 8px;
  font-weight: bold;
`
