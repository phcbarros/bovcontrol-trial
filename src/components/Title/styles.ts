import styled from 'styled-components/native'

export const Title = styled.Text`
  font-size: 24px;
  font-weight: bold;
  color: ${({theme}) => theme.colors.title};
  margin-top: 20px;
`
