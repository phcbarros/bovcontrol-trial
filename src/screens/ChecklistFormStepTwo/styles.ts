import styled from 'styled-components/native'

export const Container = styled.View`
  flex: 1;
  background-color: ${({theme}) => theme.colors.backgroundPrimary};
  padding: 20px;
  justify-content: flex-start;
  align-items: center;
  gap: 16px;
`

export const Title = styled.Text`
  font-size: 24px;
  font-weight: bold;
  color: ${({theme}) => theme.colors.title};
  margin-top: 20px;
`
