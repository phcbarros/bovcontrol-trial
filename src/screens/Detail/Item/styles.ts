import styled from 'styled-components/native'

export const Container = styled.View`
  padding: 10px;
  border-radius: 5px;
  background-color: ${({theme}) => theme.colors.backgroundSecondary};
  margin-bottom: 10px;
`

export const Label = styled.Text`
  font-size: 16px;
  margin-bottom: 5px;
  font-weight: bold;
  color: ${({theme}) => theme.colors.title};
`

export const Value = styled(Label)`
  font-weight: normal;
`
