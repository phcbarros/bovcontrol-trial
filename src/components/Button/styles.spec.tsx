import React from 'react'
import {render} from '@testing-library/react-native'
import 'jest-styled-components'
import 'jest-styled-components/native'

import {Container, Title} from './styles'
import {ThemeProvider} from 'styled-components/native'
import theme from '../../styles/theme'

describe('Button.styles', () => {
  test('Container DEVE ser igual ao snapshot', () => {
    const componente = render(<Container />, {
      wrapper: ({children}) => (
        <ThemeProvider theme={theme}>{children}</ThemeProvider>
      ),
    }).toJSON()

    expect(componente).toMatchSnapshot()
  })

  test('Title DEVE ser igual ao snapshot', () => {
    const componente = render(<Title />, {
      wrapper: ({children}) => (
        <ThemeProvider theme={{...theme}}>{children}</ThemeProvider>
      ),
    }).toJSON()

    expect(componente).toMatchSnapshot()
  })
})
