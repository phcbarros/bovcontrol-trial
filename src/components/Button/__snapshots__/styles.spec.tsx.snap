// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`Button.styles Container DEVE ser igual ao snapshot 1`] = `
<View
  accessibilityState={
    {
      "busy": undefined,
      "checked": undefined,
      "disabled": undefined,
      "expanded": undefined,
      "selected": undefined,
    }
  }
  accessibilityValue={
    {
      "max": undefined,
      "min": undefined,
      "now": undefined,
      "text": undefined,
    }
  }
  accessible={true}
  collapsable={false}
  focusable={false}
  onClick={[Function]}
  onResponderGrant={[Function]}
  onResponderMove={[Function]}
  onResponderRelease={[Function]}
  onResponderTerminate={[Function]}
  onResponderTerminationRequest={[Function]}
  onStartShouldSetResponder={[Function]}
  style={
    {
      "alignItems": "center",
      "backgroundColor": "#8257e5",
      "height": 56,
      "justifyContent": "center",
      "opacity": 1,
      "width": "100%",
    }
  }
/>
`;

exports[`Button.styles Title DEVE ser igual ao snapshot 1`] = `
<Text
  style={
    {
      "color": "#fff",
      "fontSize": 16,
      "fontWeight": "bold",
    }
  }
/>
`;
